modules/ossm-migrating-reference-smcp-configurations.adoc

```diff
@@ -0,0 +1,450 @@
+| spec.values.pilot.env
+| spec.runtime.components.container.imageName
+| spec.values.pilot.image
+| spec.runtime.components.container.imagePullPolicy
+| spec.values.global.imagePullPolicy
+| spec.runtime.components.container.imagePullSecrets
+| spec.values.global.imagePullSecrets
+| spec.runtime.components.container.imageRegistry
+| spec.values.global.hub
+| spec.runtime.components.container.imageTag
+| spec.values.pilot.tag
+| spec.runtime.components.container.resources
+| spec.values.pilot.resources
+|===
+
+=== Deployment configuration fields
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.runtime.components.deployment.autoScaling.enabled
+| spec.values.pilot.autoscaleEnabled
+| spec.runtime.components.deployment.autoScaling.maxReplicas
+| spec.values.pilot.autoscaleMax
+| spec.runtime.components.deployment.autoScaling.minReplicas
+| spec.values.pilot.autoscaleMin
+| spec.runtime.components.deployment.autoScaling.targetCPUUtilizationPercentage
+| spec.values.pilot.cpu.targetAverageUtilization
+| spec.runtime.components.deployment.replicas
+| spec.values.pilot.replicaCount
+| spec.runtime.components.deployment.strategy.rollingUpdate.maxSurge
+| spec.values.pilot.rollingMaxSurge
+| spec.runtime.components.deployment.strategy.rollingUpdate.maxUnavailable
+| spec.values.pilot.rollingMaxUnavailable
+|===
+
+=== Pod configuration fields
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.runtime.components.pod.affinity
+| spec.values.pilot.affinity
+| spec.runtime.components.pod.affinity.nodeAffinity
+| spec.values.pilot.affinity.nodeAffinity
+| spec.runtime.components.pod.affinity.podAffinity
+| spec.values.pilot.affinity.podAffinity
+| spec.runtime.components.pod.affinity.podAntiAffinity
+| spec.values.pilot.affinity.podAntiAffinity
+| spec.runtime.components.pod.metadata.annotations
+| spec.values.pilot.podAnnotations
+| spec.runtime.components.pod.metadata.labels
+| spec.values.pilot.podLabels
+| spec.runtime.components.pod.nodeSelector
+| spec.values.pilot.nodeSelector
+| spec.runtime.components.pod.tolerations
+| spec.values.pilot.tolerations
+|===
+
+=== Defaults configuration fields
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+
+| spec.runtime.defaults.container.imagePullPolicy
+| spec.values.global.imagePullPolicy
+| spec.runtime.defaults.container.imagePullSecrets
+| spec.values.global.imagePullSecrets
+| spec.runtime.defaults.container.imageRegistry
+| spec.values.global.hub
+| spec.runtime.defaults.container.imageTag
+| spec.values.global.tag
+| spec.runtime.defaults.container.resources
+| spec.values.global.defaultResources
+| spec.runtime.defaults.deployment.podDisruption.enabled
+| spec.values.global.defaultPodDisruptionBudget.enabled
+| spec.runtime.defaults.pod.nodeSelector
+| spec.values.global.defaultNodeSelector
+| spec.runtime.defaults.pod.priorityClassName
+| spec.values.global.priorityClassName
```

Comment:
Please remove this "spec.values.global.priorityClassName"  value, it is deprecated.